build.gradle

```diff
@@ -18,6 +18,7 @@ dependencies {
     compile group: 'org.apache.poi', name: 'poi', version: '4.1.2'
     compile group: 'org.apache.poi', name: 'poi-ooxml', version: '4.1.2'
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version:'2.11.2'
+    compile group: 'org.assertj', name: 'assertj-swing-junit', version:'3.9.2'
```

Comment:
It shpuld be testCompile instead of compile as otherwise it brings assertj as a dependency to kdb-studio